wemake_python_styleguide/violations/best_practices.py

```diff
@@ -69,6 +69,7 @@
    StringConstantRedefinedViolation
    IncorrectExceptOrderViolation
    FloatKeyViolation
+   MethodDecoratorUsedForFunctionViolation
```

Comment:
This should be located in `oop.py`, I think that it is the most semantical place.